### DISCUSSION TITLE
Update 1 -> 2: Documentation unclear / unprecice

### DISCUSSION THREAD

**Discussion by Marenz** [Kozea/Radicale#1436] [Q&A general]
On https://radicale.org/1to2/ it says

```
the command line argument --export-storage was added to Radicale 1.1.x. Start Radicale 1.x.x as you would normally do, but add the argument --export-storage path/to/empty/folder.
```

however, only starting from release 1.1.3 you actually added that command line option. 
Note also the inconsistent use of 1.x.x vs 1.1.x.
Unfortunately, release 1.1.1 is the latest 1.x.x release available in Debian, leaving me so far unable to migrate my data.

Then you write 
```
You can install version 1.1.6 with:

$ python3 -m pip install --upgrade radicale==1.1.6
```
So I am thinking I maybe can use that to temporary have a radicale I can run to migrate, but this just installs a bunch of python files in the dist folder, none of which I seem to be able to run as a normal binary like I am used to.

At this point I don't really know how I can migrate my data anymore.

**Comment by Marenz** [comment]
Alright, I finally did manage to migrate my data by firing up an old debian stretch in docker, doing the pip update command and then running `python2 $(which radicale)` for migration.. but this cost me a lot of headache and time to get there.. it would be nice if this weren't so difficult to do :)

**Comment by nkduufk** [comment]
I ran into the same questions. The binary files are only generated if the mentioned command above is executed as sudo.

**Comment by c-janousek** [comment]
I don't understand why `--export-storage` has not been kept in future releases. Surely that makes more sense than requiring users to download old versions?

**Comment by Unrud** [comment]
> I don't understand why `--export-storage` has not been kept in future releases. Surely that makes more sense than requiring users to download old versions?

The newer version don't support the storage format and storage plugins of the old version.
